Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -780,6 +780,13 @@ double GridTrackSizingAlgorithm::findFrUnitSize(const GridSpan& tracksSpan, Layo
 
 void GridTrackSizingAlgorithm::computeGridContainerIntrinsicSizes()
 {
+    if (m_direction == ForColumns && m_strategy->isComputingSizeContainment()) {
+        if (auto size = m_renderGrid->explicitIntrinsicInnerLogicalSize(m_direction)) {
+            m_minContentSize = m_maxContentSize = size.value();
```

Comment:
use the WebKit style here, please (and at all the other places)
m_minContentSize = size.value();
m_maxContentSize = size.value();